docs/zh/client/java-sdk-user-guide.md

```diff
@@ -398,6 +400,94 @@ apollo.label=YOUR-APOLLO-LABEL
 3. 通过`app.properties`配置文件
     * 可以在`classpath:/META-INF/app.properties`指定`apollo.override-system-properties=true`
 
+#### 1.2.4.10 ConfigMap缓存设置
+
+> 适用于2.4.0及以上版本
+
+在2.4.0版本开始，客户端在Kubernetes环境下的可用性得到了加强，开启configMap缓存后，客户端会将从服务端拉取到的配置信息在configMap中缓存一份，在服务不可用，或网络不通，且本地缓存文件丢失的情况下，依然能从configMap恢复配置。以下是相关配置
+
+> 由于需要对configmap进行读写操作，所以客户端所在pod必须有相应读写权限，具体配置方法可参考下文
+
+`apollo.cache.kubernetes.enable`：是否启动configMap缓存机制，默认false
+
+`apollo.configmap-namespace`：将使用的configMap所在的namespace（Kubernetes中的namespace），默认值为"default"
```

Comment:
Is it better to use the same prefix?
```suggestion
`apollo.cache.kubernetes.configmap-namespace`：将使用的configMap所在的namespace（Kubernetes中的namespace），默认值为"default"
```

Reply:
Thanks a lot！I have changed it to this way